packages/httpsnippet-client-api/test/__datasets__/operationid-with-underscores/output.js

```diff
@@ -1,5 +1,5 @@
 const sdk = require('api')('https://api.example.com/operationid-with-underscores.json');
 
-sdk.anythingOperation()
```

Comment:
The `operationId` for this test is/was `anything_Operation`, which is totally fine as an SDK method.